static/app/views/dashboardsV2/widgetBuilder/releaseWidget/fields.tsx

```diff
@@ -1,18 +1,87 @@
-import {MetricsMeta, MetricsOperation, SelectValue} from 'sentry/types';
+import {
+  SelectValue,
+  SessionAggregationColumn,
+  SessionsMeta,
+  SessionsOperation,
+} from 'sentry/types';
 import {defined} from 'sentry/utils';
-import {METRICS_OPERATIONS} from 'sentry/utils/metrics/fields';
 import {FieldValue, FieldValueKind} from 'sentry/views/eventsV2/table/types';
 
+export const SESSIONS_FIELDS: Record<string, SessionsMeta> = {
+  session: {
+    name: 'session',
+    operations: ['sum'],
+    type: 'integer',
+  },
+  user: {
+    name: 'user',
+    operations: ['count_unique'],
+    type: 'string',
+  },
+  'session.duration': {
+    name: 'session.duration',
+    operations: ['avg', 'p50', 'p75', 'p95', 'p99', 'max'],
+    type: 'duration',
+  },
+};
+
+export const SESSIONS_OPERATIONS: Readonly<
+  Record<SessionsOperation, SessionAggregationColumn>
+> = {
+  sum: {
+    columnTypes: ['integer'],
+    defaultValue: 'session',
+    outputType: 'integer',
+  },
+  count_unique: {
+    columnTypes: ['string'],
+    defaultValue: 'user',
+    outputType: 'integer',
+  },
+  avg: {
+    columnTypes: ['duration'],
+    defaultValue: 'session.duration',
+    outputType: null,
+  },
+  max: {
+    columnTypes: ['duration'],
+    defaultValue: 'session.duration',
+    outputType: null,
+  },
+  p50: {
+    columnTypes: ['duration'],
+    defaultValue: 'session.duration',
+    outputType: null,
+  },
+  p75: {
+    columnTypes: ['duration'],
+    defaultValue: 'session.duration',
+    outputType: null,
+  },
+  p95: {
+    columnTypes: ['duration'],
+    defaultValue: 'session.duration',
+    outputType: null,
+  },
+  p99: {
+    columnTypes: ['duration'],
+    defaultValue: 'session.duration',
+    outputType: null,
+  },
+};
+
+export const SESSIONS_TAGS = ['environment', 'project', 'release', 'session.status'];
+export const SESSION_STATUSES = ['all', 'healthy', 'abnormal', 'crashed', 'errored'];
```

Comment:
Should we apply this enum to tighten the types?
https://github.com/getsentry/sentry/blob/496e21347d491f2d06bc0b6eac5a08c9b2ec00a5/static/app/types/organization.tsx#L231-L236